src/MAVSDKConfig.cmake.in

```diff
@@ -2,13 +2,13 @@ include(CMakeFindDependencyMacro)
 
 if(NOT @BUILD_SHARED_LIBS@)
     if(${CMAKE_VERSION} VERSION_LESS "3.9.0")
-        find_package(CURL REQUIRED CONFIG)
+        find_package(CURL REQUIRED CONFIG PATHS @DEPS_INSTALL_PATH@ NO_DEFAULT_PATH)
```

Comment:
Doesn't `NO_DEFAULT_PATH` break everything that is not coming from our superbuild? What if I handled the dependencies manually and used `-DSUPERBUILD=OFF`? :thinking: